lib/commons/aria/roles.js

```diff
@@ -6,15 +6,15 @@
  * @memberof axe.commons.aria
  * @instance
  * @param  {String}  role The role to check
+ * @param  {Object}  options Use `allowAbstract` if you want abstracts
  * @return {Boolean}
  */
-aria.isValidRole = function(role) {
-	'use strict';
-	if (aria.lookupTable.role[role]) {
-		return true;
+aria.isValidRole = function(role, { allowAbstract } = {}) {
```

Comment:
The description of this issue says `disallowAbstract`, while this says `allowAbstract`. Shouldn't they be the same?